src/components/TaskStatusIcon/index.tsx

```diff
@@ -10,39 +10,45 @@ const failurePurple = "#36367F";
 interface TaskStatusIconProps {
   status: string;
   size?: Size | number;
+  style?: React.CSSProperties;
```

Comment:
Unfortunately wrapping `<Icon />` with `styled` wouldn't apply the generated className to the component even though wrapping `Icon` does.